config.py

```diff
@@ -0,0 +1,12 @@
+# omni
+# TELEGRAM_BOT_TOKEN = "5925005582:AAGmVJKdFfgjajayG2XJUMk2y3iRLPCOVME"
+# shopify
+TELEGRAM_BOT_TOKEN = "5846396253:AAH-zzCV3XuoqAJDFeWg7Po_lh41jT30PMc"
+# free version
+CHAT_GPT_API_KEY = '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'
```

Comment:
you need to remove it before pushing changes

```suggestion
CHAT_GPT_API_KEY = ''
```